src/promptflow-tools/promptflow/tools/list.py

```diff
@@ -1,13 +1,14 @@
 from pathlib import Path
 
-import yaml
+from ruamel.yaml import YAML
```

Comment:
We need to add changelog for this change. So remind us and user as it could have potential behavior change. 

ruamel.yaml may be more strict than yaml.safe_load in some cases

Reply:
Updated.